examples/async-multiplexed.rs

```diff
@@ -0,0 +1,50 @@
+extern crate futures;
+extern crate redis;
+use futures::{future, prelude::*};
+use redis::{aio::MultiplexedConnection, RedisResult};
+
+fn test_cmd(con: &MultiplexedConnection, i: i32) -> impl Future<Output = RedisResult<()>> + Send {
```

Comment:
```suggestion
async fn test_cmd(con: &MultiplexedConnection, i: i32) -> RedisResult<()> {
```

The remove the `async move { }` block in the function and it should work

Reply:
Right, went down this road.  Pushed the change which does not compile with:
```
cannot infer an appropriate lifetime for borrow expression due to conflicting requirements

note: ...so that closure can access `con`
```
Hoping that lifetime syntax does not need to be introduced here.  Is there a simple way out?